sympy/stats/stochastic_process_types.py

```diff
@@ -602,8 +640,9 @@ class DiscreteMarkovChain(DiscreteTimeStochasticProcess, MarkovProcess):
     >>> T = Matrix([[0.5, 0.2, 0.3],[0.2, 0.5, 0.3],[0.2, 0.3, 0.5]])
     >>> Y = DiscreteMarkovChain("Y", [0, 1, 2], T)
     >>> YS = DiscreteMarkovChain("Y")
+
     >>> Y.state_space
-    FiniteSet(0, 1, 2)
+    (0, 1, 2)
```

Comment:
I see, so, `state_index` is something else, publicly, there isn't any affect on the APIs. Fine then.